src/sandstorm/util.c++

```diff
@@ -601,6 +601,18 @@ kj::Array<byte> base64Decode(kj::StringPtr input) {
 
 // =======================================================================================
 
+kj::String bytesToHex(kj::ArrayPtr<const byte> input) {
```

Comment:
We actually already have hexEncode() in sandstorm-http-bridge.c++. Perhaps move that out to util.{h,c++}. (It's shorter.)